<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleWindow</class>
 <widget class="QMainWindow" name="TitleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Utah Wildlife Simulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>362</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="titleOptionsLayout">
     <item>
      <widget class="QPushButton" name="practiceButton">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Assets/Graphics/TitleButtons/practiceButton.png</normaloff>:/Assets/Graphics/TitleButtons/practiceButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adventureButton">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Assets/Graphics/TitleButtons/adventureButton.png</normaloff>:/Assets/Graphics/TitleButtons/adventureButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="shortcut">
        <string>3</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="learnButton">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Assets/Graphics/TitleButtons/learnButton.png</normaloff>:/Assets/Graphics/TitleButtons/learnButton.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>360</width>
         <height>60</height>
        </size>
       </property>
       <property name="shortcut">
        <string>1</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>385</width>
      <height>170</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Assets/Graphics/TitleButtons/Logo.png);
image: url(:/Assets/Graphics/TitleButtons/Logo.png);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Assets/Graphics/TitleButtons/infoButton.png</normaloff>:/Assets/Graphics/TitleButtons/infoButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Assets/Graphics/TitleButtons/TTMLogo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logoTextLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Brought to
you by:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>infoButton</zorder>
   <zorder>logoLabel</zorder>
   <zorder>logoTextLabel</zorder>
   <zorder>welcomeLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
